src/libraries/System.Net.HttpListener/src/System/Net/Windows/HttpListener.Windows.cs

```diff
@@ -132,7 +132,7 @@ internal void SetServerTimeout(int[] timeouts, uint minSendBytesPerSecond)
 
             SetUrlGroupProperty(
                 Interop.HttpApi.HTTP_SERVER_PROPERTY.HttpServerTimeoutsProperty,
-                infoptr, (uint)Marshal.SizeOf(typeof(Interop.HttpApi.HTTP_TIMEOUT_LIMIT_INFO)));
+                infoptr, (uint)Marshal.SizeOf<Interop.HttpApi.HTTP_TIMEOUT_LIMIT_INFO>());
```

Comment:
These are blittable structs and they are passed in into the PInvoke without any marshalling. It would be better to replace this with `sizeof(Interop.HttpApi.HTTP_TIMEOUT_LIMIT_INFO)`. Similar in other places

Reply:
While you are on it - could you please also delete `infoptr` local, pass `&timeoutinfo` directly to the `HttpServerTimeoutsProperty` method, and replace `IntPtr` with `void*` in the implementation of the `HttpServerTimeoutsProperty` method?